examples/api/demo_affine_image.py

```diff
@@ -24,22 +21,15 @@ def get_image():
     return Z
 
 
-def imshow_affine(ax, z, *kl, **kwargs):
-    im = ax.imshow(z, *kl, **kwargs)
-    x1, x2, y1, y2 = im.get_extent()
-    im._image_skew_coordinate = (x2, y1)
-    return im
-
-
 if 1:
 
     # image rotation
 
-    fig, (ax1, ax2) = plt.subplots(1, 2)
+    fig, ax1 = plt.subplots(1, 1)
     Z = get_image()
-    im1 = imshow_affine(ax1, Z, interpolation='none', cmap=cm.jet,
-                        origin='lower',
-                        extent=[-2, 4, -3, 2], clip_on=True)
+    im1 = ax1.imshow(Z, interpolation='none', cmap=cm.jet,
```

Comment:
I'm fine with removing this now, or letting it be part of #5774, which removes jet in all the examples.